java/pom.xml

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>GraphBulkExecutor</artifactId>
+    <version>1.0</version>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <cosmosSDKVersion>4.24.0</cosmosSDKVersion>
+        <lombokVersion>1.18.22</lombokVersion>
+        <commonsVersion>3.12.0</commonsVersion>
+        <snapshotDependencyAllowed>true</snapshotDependencyAllowed>
+        <skipIntegrationTests>true</skipIntegrationTests>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>${lombokVersion}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.azure</groupId>
+            <artifactId>azure-cosmos</artifactId>
+            <version>${cosmosSDKVersion}</version>
```

Comment:
Instead of providing an `azure-cosmos` version, can we not always rely on the latest? So basically skipping the version here at all. Something like this - https://github.com/Azure-Samples/azure-cosmos-java-sql-api-samples/blob/main/pom.xml#L48

Otherwise we would have to push updates to this regularly, and creates overhead. 
Also, when customers copy paste, they will stick to version what we use in the samples, so best to always use latest.